stories/topics/proc-aws-from-stack-pull-container-image.adoc

```diff
@@ -7,24 +7,25 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, ensure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
+Before pulling the docker image, ensure you are logged in to {Registry} using docker. Use the following command to login to {Registry}.
```

Comment:
@ianf77 Is this procedure used in any assemblies?

Reply:
Possibly not, but it wasn't archived.
There's a gcp file proc-gcp-pull-backup-container-image which seems to have the wrong image. Either they missed it or it's not used. I can't find the value in the docs.